src/App.jsx

```diff
@@ -1,7 +1,27 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+
+import RegionsContainer from './RegionsContainer';
+
+import { setRegions } from './actions';
+
+function loadRegions({ dispatch }) {
+  const regions = [];
+
+  dispatch(setRegions(regions));
+}
 
 export default function App() {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    loadRegions({ dispatch });
+  }, {});
```

Comment:
```suggestion
  useEffect(() => {
    loadRegions({ dispatch });
  }, []);
```
최초 한번만 dispatch 하려는 지금과 같은 상황에는 두번째 인자로 빈배열을 넘겨줘야합니다.
- https://ko.reactjs.org/docs/hooks-reference.html#conditionally-firing-an-effect